client/src/elements/schema-editor/schema-editor-files.js

```diff
@@ -103,6 +104,20 @@ export class SchemaEditorFiles {
       this.dropzoneOptions
     );
 
+    this.dropzone.on("sending", (file, xhr, data) => {
+      if (
+        this.options.pathPrefix !== null &&
+        this.options.pathPrefix !== undefined
+      ) {
+        if (file.fullPath) {
```

Comment:
Lets rename `fullPath` to `fileKey`. Q-Server also names is like this. See code here -> https://github.com/nzzdev/Q-server/blob/dev/plugins/file/index.js#L86